fc/jira/issue.py

```diff
@@ -0,0 +1,141 @@
+            'fields': {
+                'project': {
+                    'key': 'QPPFC'
+                },
+                'summary': self.title,
+                'description': self.description,
+                'components': [{
+                    'name': 'Foundational'
+                }]
+            }
+        }
+
+        self._extra_json_for_create(json)
+
+        response = requests.post(self.api_url, json=json,
+                                 auth=HTTPBasicAuth(self.auth.username(), self.auth.password()))
+        response.raise_for_status()
+
+        response_json = response.json()
+
+        self.id = response_json['key']
+        self.url = self.base_url.format(self.id)
+
+        return self.id, self.url
+
+    def _extra_json_for_create(self, existing_json: dict):
+        raise NotImplementedError
+
+    def type_str(self) -> str:
+        raise NotImplementedError
+
+    @classmethod
+    def _get_issue(cls, issue_id: str, auth: Auth) -> dict:
+        response = requests.get(cls.api_url + issue_id,
+                                auth=HTTPBasicAuth(auth.username(), auth.password()))
+        response.raise_for_status()
+
+        return response.json()
+
+    def _get_active_sprint_id_of_issue(self, issue_id: str) -> Optional[int]:
+        issue = self._get_issue(issue_id, self.auth)
+
+        sprint_list = issue['fields'][self.issue_assigned_sprint_field]
+
+        if sprint_list is None:
+            return None
+
+        active_sprint = sprint_list[-1]
+        id_begin_token = 'id='
+        id_begin = active_sprint.find(id_begin_token) + len(id_begin_token)
+        id_end = active_sprint.find(',', id_begin)
+
+        return int(active_sprint[id_begin:id_end])
+
+    def _get_transition_dict(self) -> dict:
+        raise NotImplementedError
+
+    @classmethod
+    def get_issue(cls, issue_id: str, auth: Auth) -> 'Task':
+        from .backlog_task import BacklogTask
+        from .triage_task import TriageTask
+        from .el_task import ElTask
+
+        issue_json = {}
+        task = None
+
+        try:
+            issue_json = cls._get_issue(issue_id, auth)
+        except HTTPError as exception:
+            raise TaskException('Invalid issue key {}'.format(issue_id)) from exception
+
+        type = issue_json['fields']['issuetype']['name']
+        if type != 'Task' and type != 'Triage Task':
+            raise TaskException('Invalid issue type {}'.format(type))
+        elif type == 'Task':
+            task = BacklogTask.from_json(issue_json, auth)
+        else:
+            labels: list = issue_json['fields']['labels']
+            if 'EL' in labels:
+                task = ElTask.from_json(issue_json, auth)
+            else:
+                task = TriageTask.from_json(issue_json, auth)
+
+        return task
+
+    def comment(self, note: str):
+
+        json = {
+            'body': note
+        }
+
+        response = requests.post(self.api_url + self.id + '/comment', json=json,
```

Comment:
```suggestion
        response = requests.post('{}{}/comment'.format(self.api_url, self.id), json=json,
```
String concatenation, in general, is frowned upon.